torch_tvm/custom_tvm_ops/topi/topi_init.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+namespace tvm {
+void registerCustomTopiSchedules();
```

Comment:
For OSS, python oriented user this might be a problem.

Reply:
is there a pythonic way of registering relay compute?

Reply:
Yeah, but what I'm thinking is why not upstreaming this to TVM directly?

Reply:
I'm not sure about that -- @kimishpatel ?

Reply:
I am ok with upstreaming it. We can discuss this.